src/params/screening.jl

```diff
@@ -0,0 +1,35 @@
+struct ScreeningParams
+    begin_day::Float64
+    precision::Float64
+    frequency::Float64
+  end
+
+
+function screen_children!(rng::AbstractRNG, state::SimState, params::SimParams)
+    for screening_day in params.screening_params.begin_day:params.screening_params.frequency:365
```

Comment:
design: but maybe it should not be a function of ::SimParams
- this does not check if `params.screening === nothing`.
- 365 is a hardcoded value

Reply:
I removed hardcoded value and I added check, but fuction is still a part ::SimParams